assembl/static2/js/app/components/debate/common/post/postBody.jsx

```diff
@@ -63,15 +63,15 @@ const PostBody = ({
   translate,
   translationEnabled
 }: Props) => (
-  <div className="post-body">
+  <div className={`post-body${translate ? '' : ' post-body--is-harvestable'}`}>
```

Comment:
similarly, `classnames` is your friend. 

https://www.npmjs.com/package/classnames